WireDomain/Sources/WireDomain/Event Decryption/UpdateEventDecryptor.swift

```diff
@@ -122,6 +122,10 @@ struct UpdateEventDecryptor: UpdateEventDecryptorProtocol {
             }
         }
 
+        Task.detached {
+            await mlsMessageDecryptor.commitPendingProposalsIfNeeded()
+        }
```

Comment:
suggestion: leave a comment here stating that this may take a long time and therefore we don't want to wait for it.

Reply:
done https://github.com/wireapp/wire-ios/pull/2685/commits/364d6d4db0edc47cfb6b9c632fbbcac8f6ab2393